DEVELOPER_GUIDE.md

```diff
@@ -21,7 +22,7 @@ Read more about extensibility [here](https://github.com/opensearch-project/OpenS
 Fork [OpenSearch SDK](https://github.com/opensearch-project/opensearch-sdk) and clone locally, e.g. `git clone https://github.com/[your username]/opensearch-sdk.git`.
 
 ### Git Clone OpenSearch Repo
-Fork [OpenSearch](https://github.com/opensearch-project/OpenSearch) and clone locally, e.g. `git clone https://github.com/[your username]/OpenSearch.git`.
+Fork [OpenSearch](https://github.com/opensearch-project/OpenSearch/), checkout feature/extensions branch, and clone locally, e.g. `git clone https://github.com/[your username]/OpenSearch.git`.
```

Comment:
Wouldn't you clone it locally and then checkout the branch?  When I fork on GitHub it forks all the branches and there's not really a "checkout" feature there.

Reply:
Yeah that's correct I'll update it

Reply:
Actually I just noticed that the checking out step is already in the next section, so I'll just delete the part I added